src/tools/TimeTools.cpp

```diff
@@ -60,9 +60,13 @@ std::string timeTools::convertUnixTimestampToIso(const long long & ts)
 time_t timeTools::convertIsoToUnixTimestamp(const std::string & s)
 {
 	struct tm tm;
+#if defined(__gnu_linux__) || defined(__APPLE__)
```

Comment:
I think it is more a check on C++11 than on the OS : https://stackoverflow.com/questions/10717502/is-there-a-preprocessor-directive-for-detecting-c11x-support

Reply:
C++11 will be the minimum version for the coming FESAPI ETP version. We will resolve that at this time.